README.md

```diff
@@ -1,123 +1,91 @@
+[bundletester](https://github.com/juju-solutions/bundletester).
 
-Note: jujubox does not currently install juju local provider.  We hope
-to roll something that works like local provider in a container
-solution soon.
+# Install Docker
 
-# Install
+## Linux
+Install Docker using the package manager for your Linux distribution, or
+[get.docker.com](https://get.docker.com/). We recommend Docker version 1.4.1 or
+greater.
 
-### Linux
-Install Docker using the package manager for your distro, or
-[get.docker.com](https://get.docker.com/).
+## Mac OS X
 
-Docker >= 1.4.1 recommended.
+There are several ways to 
+[install Docker on Mac OS X](https://docs.docker.com/engine/installation/mac/). 
+Consult the latest documentation on
+[Getting Started with Docker for Mac](https://docs.docker.com/docker-for-mac/).
 
-### OS X
+# Download jujubox
 
-Docker does not run natively in OS X, so we'll need to use VirtualBox
-and boot2docker. This is relatively easy for users of brew:
-
-```
-brew install cask
-brew cask install virtualbox
-brew install docker
-brew install boot2docker
-
-boot2docker init
-boot2docker up
-```
-
-When boot2docker finishes, it will prompt you to export Docker
-environment variables. With that done, you're ready to use docker.
-You do not need to run the docker commands using sudo because
-boot2docker runs as root.
-
-    docker run  -ti jujusolutions/jujubox
-
-# Download
-
-Download the Juju Box source from github or the container from
-[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/):
-
-### Linux
-You can clone the repository from github.
+If you want to see the code you can download the jujubox project from 
+[github](https://github.com/juju-solutions/jujubox) and build it from source.
 
 ```
 git clone https://github.com/jujusolutions/jujubox.git && cd jujubox
-
+docker build -t jujusolutions/jujubox ./
 ```
 
-Or download and install the container directly from hub.docker.io:
+Or simply pull the jujubox container from
+[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/).
 
 ```
-sudo docker run  -ti jujusolutions/jujubox
+docker pull jujusolutions/jujubox
 ```
 
+# Run jujubox
 
-# To Run
+There are several different ways to run jujubox depending on what you are 
+trying to accomplish. 
 
-There are two ways to run the docker container, from the github
-repository (build from source) or from the container directly from the
-hub.docker.io.
+## Running jujubox to evaluate Juju
 
+If you do not already have Juju installed and want to check this container out 
+you can simply run the container in an interactive (-i) tty (-t) session to use
+the Juju tools:
 
-### Linux
-
-#### From source:
 ```
-sudo docker build -t jujubox ./
-sudo docker run -i -t jujubox:latest
+docker run --rm -it jujusolutions/jujubox 
 ```
 
-#### Use hub.docker.io:
-```
-sudo docker run  -ti jujusolutions/jujubox
-```
+## Running jujubox if you have Juju installed on host
```

Comment:
I didn't get this far down, totally think this should be merged with the documentation paragraph above. Again, orphaning machines in the cloud can be an expensive proposition.